### ISSUE TITLE
An error occurred when this program started running.

### ISSUE THREAD

**Issue by a65090482** [gtasa-savegame-editor/gtasa-savegame-editor#105]
I installed JAVA (and its version): Java(TM) Development Kit 11.0.2 (64-bit)
Specific text about the error: 

```
java.lang.reflect.InvocationTargetException
	at java.desktop/java.awt.EventQueue.invokeAndWait(EventQueue.java:1367)
	at java.desktop/java.awt.EventQueue.invokeAndWait(EventQueue.java:1342)
	at java.desktop/javax.swing.SwingUtilities.invokeAndWait(SwingUtilities.java:1480)
	at nl.paulinternet.gtasaveedit.view.Main.main(Main.java:68)
Caused by: java.lang.NullPointerException
	at nl.paulinternet.gtasaveedit.view.component.TabbedPane.updateUI(TabbedPane.java:110)
	at java.desktop/javax.swing.JTabbedPane.<init>(JTabbedPane.java:207)
	at java.desktop/javax.swing.JTabbedPane.<init>(JTabbedPane.java:172)
	at nl.paulinternet.gtasaveedit.view.component.TabbedPane.<init>(TabbedPane.java:26)
	at nl.paulinternet.gtasaveedit.view.window.MainWindow.createContent(MainWindow.java:49)
	at nl.paulinternet.gtasaveedit.view.GUICreator.run(GUICreator.java:23)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:303)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at nl.paulinternet.gtasaveedit.view.EventQueueProxy.dispatchEvent(EventQueueProxy.java:14)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

```

**Comment by a65090482** [comment]
Is there a way to deal with this error?

**Comment by a65090482** [comment]
Is it necessary to deal with this error?

**Comment by lfuelling** [comment]
Hi, 

I'll fix this bug later today (it was introduced while I tried to hotfix #101).

If you need a working version **now**, you can use [`v3.3-beta.5.1`](https://github.com/gtasa-savegame-editor/gtasa-savegame-editor/releases/tag/v3.3-beta.5.1) which should work and create valid savegames.

**Comment by a65090482** [comment]
Thank you. May this project be better and better.

**Comment by lfuelling** [comment]
Closed with #120
